packages/next/package.json

```diff
@@ -101,6 +101,7 @@
     "styled-jsx": "5.1.1"
   },
   "peerDependencies": {
+    "@playwright/test": "^1.41.2",
```

Comment:
This needs to be marked as optional like below for `@opentelemetry/api`

Reply:
Changes made